src/main.cpp

```diff
@@ -53,8 +53,11 @@ int main(int argc, char* argv[])
     int n_pi_bins = 1000;           // No. of WSAF bins in betabinomial lookup
     
     // MCMC parameters
-    double w_proposal_sd = 0.1;     // Titres sampled from ~N(0, w_propsal_sd)
-    int n_temps = 5;                // Number of temperature levels for PT-MCMC
+    int n_burn_iters = 100;              // Number of iterations in burn-in phase
+    int n_sample_iters = 900;            // Number of iterations in sampling phase
+    int n_temps = 5;                     // Number of temperature levels for PT-MCMC
+    double target_acceptance = 0.44;     // Target acceptance rate per MCMC rung
```

Comment:
This maybe doesn't need to be a parameter, but could be a global constant? Or might we want to tune this?